content/developers/api-reference/scitt-api/index.md

```diff
@@ -0,0 +1,178 @@
+
+When registering statements about an artifact, a common identifier is required to corelate a series of statements.
+A SCITT `CWT_Claims subject` property captures the identifier.
+
+The example uses SPDX, which generates a [Document Namespace](https://spdx.github.io/spdx-spec/v2.2.2/document-creation-information/#65-spdx-document-namespace-field) field which is used for the `subject` property.
+
+_\<TODO: Add a doc for creating unique identifiers>_
+
+1. Generate an SPDX SBOM for `artifact.js`
+
+    ```bash
+    sbom-tool generate -D true \
+      -b artifacts \
+      -ps scitt-community \
+      -pn scitt-nodejs-example -pv 0.0.0 \
+      -nsb https://scitt.io/examples
+    ```
+
+    [More info for the SPDX sbom-tool](https://github.com/microsoft/sbom-tool/blob/main/docs/sbom-tool-arguments.md)
+1. Capture the Document Namespace for the SCITT `subject`
+
+    ```shell
+    SUBJECT=$(cat artifacts/_manifest/spdx_2.2/manifest.spdx.json | jq -r .documentNamespace)
+    ```
+
+1. Create a Signed Statement for the SPDX SBOM
+
+    ```shell
+    datatrails statement create \
+      issuer <identity-reference> \
+      subject $SUBJECT \
+      payload artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      content-type application/spdx+json \
+      output signed-statement.cbor
+    ```
+
+1. Register the SPDX SBOM for the artifact
+
+    ```shell
+    ENTRY_ID=$(datatrails register \
+      signed-statement signed-statement.cbor)
+    ```
+
+1. **COMBINED?:** Sign and Register the SBOM<br>
+    An alternative to the above two commands
+
+    ```shell
+    ENTRY_ID=$(datatrails statement register \
+      issuer <identity-reference> \
+      subject $SUBJECT \
+      payload artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      content-type application/spdx+json)
+    ```
+
+1. Retrieve a SCITT Receipt
+
+    ```shell
+    datatrails statement get-receipt $ENTRY_ID
+    ```
+
+## Register an Attestation
+
+Create an Attestation that provides the artifacts compliance to the \<foo> specification
+
+1. Create the Attestation
+
+    ```shell
+    cat > attestation.json <<EOF
+    {
+      "compliance": [
+        {
+          "name": "standard-alpha",
+          "value": "true"
+        },
+        {
+          "name": "standard-beta",
+          "value": "false"
+        }
+      ]
+    }
+    EOF
+    ```
+
+1. Sign and Register the Attestation
+
+    ```shell
+    ENTRY_ID=$(datatrails statement register \
+      issuer <identity-reference> \
+      subject $SUBJECT \
+      payload attestation.json \
+      content-type attestation/example )
+    ```
+
+## Retrieve Statements for the Artifact
+
+The power of SCITT is the ability to retrieve the history of statements made for a given artifact.
+By querying the series of statements, consumers can verify who did what and when for a given artifact.
+
+1. Query DataTrails for the collection of statements
+
+    ```shell
+    datatrails statement get-by-subject $SUBJECT
```

Comment:
This is a very powerful thing datatrails can do as part of its bespoke API, I like that we have that in here :)